libraries/helpers.rb

```diff
@@ -0,0 +1,65 @@
+module OslRepos
+  module Cookbook
+    module Helpers
+      # Select the gpg key for use in centos 7
+      # This ensures that the AltArch gpg key is included in any non x86_64 architecture
+      def centos_7_gpgkey
+        if node['kernel']['machine'] == 'x86_64'
+          'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7'
+        else
+          'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-AltArch-7-$basearch'
+        end
+      end
+
+      # Select the epel baseurl based on centos version
+      def epel_baseurl
+        case node['platform_version'].to_i
+        when 7
+          node.default['yum']['epel']['baseurl'] = 'https://epel.osuosl.org/$releasever/$basearch/'
+        when 8
+          node.default['yum']['epel']['baseurl'] = 'https://epel.osuosl.org/$releasever/Everything/$basearch/'
```

Comment:
We don't want to edit the attribute here as a method, we'll do that in the resource. This should just return the string we need. I noticed this why testing this on osl-haproxy when switching between the current and new way.
```suggestion
          'https://epel.osuosl.org/$releasever/$basearch/'
        when 8
          'https://epel.osuosl.org/$releasever/Everything/$basearch/'
```